packages/react-text/README.md

```diff
@@ -1,5 +1,99 @@
 # @fluentui/react-text
 
-**React Text component and wrappers for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+> WIP 🚧 - These are not production-ready components as we are still in a Beta phase.
 
-These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+<!-- TODO: Add link to the new website -->
+
+The Text component exists to ensure consistency in your application's content by setting fixed sizes and other styles.
+This package also exports wrappers which ensure your text follows the Fluent design standards of typography.
+
+## Usage
+
+To use the Text components in your application, you can start by installing the main package of Fluent UI components:
+
+<!-- TODO: Validate if FluentProvider works without theme. If not, which theme should we refer to -->
+
+```sh
+npm install @fluentui/react-components
+```
+
+```jsx
+import { FluentProvider, Text } from '@fluentui/react-components';
+
+const App = () => (
+  <FluentProvider>
+    <Text>Fluent UI Text!</Text>
+  </FluentProvider>
+);
+```
+
+Or by installing only the `react-text` package. Keep in mind you'll need to install the FluentProvider package as well:
+
+```sh
+npm install @fluentui/react-text
+npm install @fluentui/react-provider
+```
+
+```jsx
+import { FluentProvider } from '@fluentui/react-provider';
+import { Text } from '@fluentui/react-text';
+
+const App = () => (
+  <FluentProvider>
+    <Text>Fluent UI Text!</Text>
+  </FluentProvider>
+);
+```
+
+## Typography wrappers
+
+![List of typography variants by sorted descending by size](./assets/typography-examples.gif 'Typography wrapper list')
```

Comment:
why gif for static image ?

Reply:
Size optimization purposes only 😄

Reply:
haha, ok can we use jpeg  or more modern approaches ? to me this is confusing ->
"like oh `gif` ! there is gonna be some movement - but I cant see any damn...what's wrong?"

Reply:
JPEG would imply a size around 4x bigger or visual degradation to achieve smaller sizes. Would it make a big difference if it stayed GIF? It's only visible on the source anyway and irrelevant when the MD is rendered in Github.

Reply:
I think we are prematurely optimising for non existent use case. 

- github is rendering those assets, hopefully they are optimizing accordingly ?
  - only 2 violations regarding images (lighthouse report)  
    -  explicit size  ![2021-08-03 at 15 53](https://user-images.githubusercontent.com/1223799/128027620-b93cc612-1ab9-4984-941e-064580693bf8.png)
 
    - pixel ration ![2021-08-03 at 15 51](https://user-images.githubusercontent.com/1223799/128027455-cf301da3-b5c3-431c-b932-86afbfa08443.png)

 
- if we are concerned about this we should use webp or avif formats
- as seen from the report is not doable by us to optimize much what is being rendered by github ( source we dont controll)
- if this is rendered via storybook, the "issue" is even more relevant as we are shipping extensive amount of javascript (where kB up and down for static asset like image is negligible )